pkg/storage/etcd/testing/utils.go

```diff
@@ -155,7 +155,8 @@ func (m *EtcdTestServer) Terminate(t *testing.T) {
 	time.Sleep(250 * time.Millisecond)
 	for _, hs := range m.hss {
 		hs.CloseClientConnections()
-		hs.Close()
+		// TODO: Uncomment when fix #19254
+		// hs.Close()
```

Comment:
Yeah, there was broad band-aid put in in https://github.com/kubernetes/kubernetes/pull/19458 at least this is consistent with that?